src/geovista/bridge.py

```diff
@@ -768,18 +781,29 @@ def from_tiff(
 
         Examples
         --------
+        >>> from geovista import GeoPlotter, Transform
+        >>> from geovista.pantry import fetch_raster
+        >>> from geovista.pantry.textures import natural_earth_1
+
         Render the GeoTIFF ``RGB`` image as a geolocated mesh.
 
         First, :func:`rasterio.features.sieve` the GeoTIFF image to remove several
         unwanted masked regions within the interior of the image, due to a lack of
         dynamic range in the ``uint8`` image data. Then, extract the mesh to remove
         cells with no masked points.
 
-        >>> from geovista import Transform
-        >>> from geovista.pantry import fetch_raster
         >>> fname = fetch_raster("bahamas_rgb.tif")
         >>> mesh = Transform.from_tiff(fname, rgb=True, sieve=True, extract=True)
-        >>> mesh.plot(cpos="xz", lighting=False, rgb=True)
+
+        Plot the result!
+
+        >>> plotter = GeoPlotter()
+        >>> _ = plotter.add_base_layer(texture=natural_earth_1())
+        >>> _ = plotter.add_coastlines()
+        >>> _ = plotter.add_mesh(mesh, rgb=True)
+        >>> plotter.view_poi()
+        >>> plotter.camera.zoom(40)
+        >>> plotter.show()
```

Comment:
https://geovista--841.org.readthedocs.build/en/841/reference/generated/api/geovista/bridge/index.html#geovista.bridge.Transform.from_tiff

Reply:
Fun times, right ?

I'm lovin' the rich interactive docstring rendering, it's just lush 😍 

As an aside, you wouldn't fancy making the coastlines white ? i.e., `_ = plotter.add_coastlines(color="white")`